homeassistant/components/monoprice/config_flow.py

```diff
@@ -99,7 +99,9 @@ def async_get_options_flow(config_entry):
 @core.callback
 def _key_for_source(index, source, previous_sources):
     if str(index) in previous_sources:
-        key = vol.Optional(source, default=previous_sources[str(index)])
+        key = vol.Optional(
+            source, description={"suggested_value": previous_sources[str(index)]}
```

Comment:
Has `suggested_value` been documented in our developer docs?

Reply:
It hasn't. I couldn't find any place where the use of schema for config flows was documented at all, so didn't really know where to put this.
More than happy to update the docs to reflect this if there's a relevant place.

Reply:
Please document it here:
https://developers.home-assistant.io/docs/data_entry_flow_index#show-form

Reply:
https://github.com/home-assistant/developers.home-assistant/pull/492